dev-python/pyct/pyct-0.4.8.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Python packaging Common Tasks"
+HOMEPAGE="
+	https://pypi.org/project/pyct
+	https://github.com/pyviz-dev/pyct"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=">=dev-python/param-1.7.0[${PYTHON_USEDEP}]"
```

Comment:
Definitely needed at build time?

Reply:
Definitely. I solemnly swear it on my fat Maine Coon cat.